rocketpy/plots/aero_surface_plots.py

```diff
@@ -380,6 +380,67 @@ def draw(self):
         plt.show()
 
 
+class _FreeFormFinsPlots(_FinsPlots):
+    """Class that contains all free form fin plots."""
+
+    # pylint: disable=too-many-statements
+    def draw(self):
+        """Draw the fin shape along with some important information, including
+        the center line, the quarter line and the center of pressure position.
+
+        Returns
+        -------
+        None
+        """
```

Comment:
The draw method is not closing the fin draw when the last point is different than the initial point, as shown in the figure below

![image](https://github.com/user-attachments/assets/0bdc8ca2-7fc8-491e-ad14-926b43137bc4)

Reply:
how are we going to close it? linearly? we should provide an example (in a future PR maybe) showing case when the fin is set at a non-planar surface. for example when the fins are placed on the tail/transition

Reply:
Well, I assumed that the Fin is defined by a closed polygon. Indeed, the documentation states that "the last point should be the first one." So, if the user "forgot" to make sure the last point is the first one, we either should raise an error or just close it ourselves.

Reply:
The model assumes that it is closed linearly. I will add the last line in the plot as well